torch/fx/passes/regional_inductor.py

```diff
@@ -141,7 +172,13 @@ def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
     marked_nodes = InductorMarkedNodes()
     gm = _partition_by_supported_nodes(gm, marked_nodes, "__marked_inductor_submod")
     gm = _compile_submod(gm, "__marked_inductor_submod")
-    return gm
+
+    gm.graph.set_codegen(_BoxedCodeGen())
+    gm.recompile()
+
+    # Use a pickleable wrapper class that provides boxed calling convention
+    wrapper = _BoxedCallWrapper(gm)
```

Comment:
hang on, why? Once you set codegen on gm, it should have boxed calling convention no?